src/core_atmosphere/mpas_atm_core.F

```diff
@@ -89,6 +89,13 @@ function atm_core_init(domain, startTimeStamp) result(ierr)
       ! input alarms for both input and restart before reading any remaining
       ! input streams.
       !
+      call MPAS_stream_mgr_read(domain % streamManager, streamID='static', whence=MPAS_STREAM_NEAREST, ierr=ierr)
+      if (ierr /= MPAS_STREAM_MGR_NOERR) then
+         call mpas_log_write('********************************************************************************', messageType=MPAS_LOG_ERR)
+         call mpas_log_write('Error reading static fields',                                                      messageType=MPAS_LOG_ERR)
+         call mpas_log_write('********************************************************************************', messageType=MPAS_LOG_CRIT)
+      end if
+      call MPAS_stream_mgr_reset_alarms(domain % streamManager, streamID='static', direction=MPAS_STREAM_INPUT, ierr=ierr)
```

Comment:
I think we should make this static stream read optional for now (e.g., through an extra namelist parameter 'separate_init_stream' or something more meaningful?), single file restart/init initialization capability should remain unchanged. It may be more acceptable for the model group if we want to commit this change to the official MPAS model repo.